examples/cert-val.diag

```diff
@@ -0,0 +1,20 @@
+ / claims-map / {
+    / comid.endorsements / 1: {
+      / element name /  0 : {
+        / implementation id / 2 : 47115(h'61636d652d696d706c656d656e746174696f6e2d69642d303030303030303031')
+      },
+
+      / comid.psa-cert-meta / 2 : {
+        / cert-level /   1 : "PSA Certified Level 1",
+        / cert-num / 2 : "1.3.5",
+        / cert-issue-date / 3 : "28/07/2020",
+        / cert-test-lab /4 : "Example Test Lab",
+        / cert-holder / 5 : "ACME Ltd",
+        / cert-product / 6 : "RoadRunner Hardware",
+        / cert-hw-version / 7 : "1.0.0",
+        / cert-sw-version / 8 : "1.0.0",
+        / cert-type / 9 : "PSA Certified Level 1 v2.1",
```

Comment:
just a curiosity: do you know what this is supposed to mean?

Reply:
I think, it gives versioning on Certification Level, but needs to check with experts!

Reply:
yes please, add it to the list of questions for the PSA Certified guys